### ISSUE TITLE
update footer values based on selected rows

### ISSUE THREAD

**Issue by bsibul16** [wenzhixin/bootstrap-table#6845]
### Description

Hi, how would I update the values in the footer based on which rows are selected? For instance, if all rows are selected and my footer is displaying the sum of values a column, when one row is deselected, I would want the footer to update so that sum does not include the value from the deselected row. How would I implement this?

**Comment by jamie-belanger** [comment]
I just ran into this and came up with a solution. I'm new to using this control, so I can't guarantee this is an optimal solution, but it does work... but in my use case, with 80-ish columns, there's some slight flickering. I'd prefer a more optimal solution. Maybe someone more experienced with Bootstrap Table can comment?

[https://live.bootstrap-table.com/code/jamie-belanger/15695](Live example)

I marked relevant lines with `// Important part` comments
